NERDweb/shodan_rpc_client.py

```diff
@@ -0,0 +1,37 @@
+import pika
+import uuid
+import codecs
+
+reader = codecs.getreader("utf-8")
+
+class ShodanRpcClient(object):
+    def __init__(self):
+        rmq_creds = pika.PlainCredentials('guest', 'guest')
+        rmq_params = pika.ConnectionParameters('localhost', 5672, '/', rmq_creds)
```

Comment:
The connection parameters should be taken from configuration (e.g. loaded in nerd_main and passed as parameters of the contructor).